Translation corrections (#301)
07th-mod/higurashi-console-arcs: Update/toki_tips_006.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>？？？</color>", NULL, "<color=#f5e6d3>???</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 32, "ps3/s20/32/vtt_hanada3056", 256, TRUE);
	OutputLine(NULL, "「どちらが菩薩で孫悟空なのか、あとで思い知らせてやりますよ。",

> 			NULL, "\"I'll show you which one of us is the Buddha and which of us is Sun Wukong.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"I'll show you which of us shall emerge a god.", 
```
